sensors/shtc3Sensor/SHTC3Sensor.cpp

```
@@ -0,0 +1,117 @@
+void SHTC3Sensor::startContinuousMeasurement() {
+    if (continuous_active_) return;
+    continuous_active_ = true;
+    xTaskCreate(continuousMeasureTask, "SHTC3MeasureTask", 2048, (void*)this, 5, nullptr);
+}
+
+void SHTC3Sensor::stopContinuousMeasurement() {
+    continuous_active_ = false;
+}
+
+void SHTC3Sensor::continuousMeasureTask(void* param) {
+    SHTC3Sensor* sensor = (SHTC3Sensor*)param;
```

Clang (14.0.6)

[warning] use auto when initializing with a cast to avoid duplicating the type name

(modernize-use-auto)